### DISCUSSION TITLE
osu!lazer crashes on startup

### DISCUSSION THREAD

**Discussion by eoyoa** [ppy/osu#15553] [Q&A] [answered]
Window does not appear, process shows up for a split second then disappears.

[performance.log](https://github.com/ppy/osu/files/7510122/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7510123/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7510124/updater.log)
[database.log](https://github.com/ppy/osu/files/7510125/database.log)
[network.log](https://github.com/ppy/osu/files/7510126/network.log)

**Comment by nahkd123** [comment, marked as answer]
See https://github.com/ppy/osu/discussions/15530
